### DISCUSSION TITLE
Changing position and size of fixed terminal?

### DISCUSSION THREAD

**Discussion by rdnrtms** [folke/snacks.nvim#336] [Q&A] [answered]
Hey folks,

I was trying to somehow set my non-floating terminal to appear from the right of the screen, instead of bottom. Also, I tried setting the height of my bottom terminal because it occupies too much space right now. Does anyone have an idea how to do it?

Thanks in advance!

**Comment by bjd000** [comment, marked as answer]
To appear on the right, set your options for terminal:
```lua 
return {
    'folke/snacks.nvim',
    priority = 1000,
    lazy = false,
    ---@type snacks.Config
    opts = {
        terminal = { 
            enabled = true,
            win= {position="right", width=0.2},
        },
...
```
Or, add it to the command:
`:lua Snacks.terminal.open(nil, { win = { position = "right" }})`

You can use 'width' and 'height' to set how much space the terminal window consumes.

**Reply by rdnrtms** [reply]
Thanks, the second solution worked better. The first solution overwrote the LazyGit floating window to open on the bottom in a new terminal-like window, so I don't recommend that for anyone reading this. I remapped the <c-/> key to 
`vim.keymap.set("n", "<c-/>", function()
    Snacks.terminal(
        nil,
        { cwd = LazyVim.root(), win = { position = "bottom", height = 0.25 } }
    )
end, { desc = "Terminal (Root Dir)" })`

Thanks again, I marked your input as answer.

**Reply by MrPandey01** [reply]
You are correct about the first solution. However your proposed second solution also doesn't work for me.
